storybookjs/storybook
# [react-native] fix drawer width

## Fix the navigation drawer width
![simulator screen shot - iphone x - 2017-11-13 at 15 35 22](https://user-images.githubusercontent.com/1374161/32731004-fa128cc2-c888-11e7-8d9a-51d83eb3145a.png)
Issue:


## What I did
The navigation drawer width was based on the `screenWidth / 2` while the **StoryListView** itself was based on a `maxWidth: 200` which could lead to the visibility of the drawer in case the 
`screenWidth / 2 < StoryListView width` 

I've just remove the `MinMaxWidth` component and put a fix width to the drawer (standard behavior).


## How to test

Is this testable with jest or storyshots? Maybe

Does this need a new example in the kitchen sink apps? No

Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
